site/content/docs/monitoring/false-positives.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Understanding and managing false positives
+weight: 10
+menu:
+  platform:
+    parent: "Monitoring"
+---
+
+At Checkly, we provide a reliable and performant solution for running synthetic monitoring and API checks across 22 regions. Our infrastructure is designed to be robust, but sometimes, unforeseen issues like server or network problems can crop up—whether on our side or yours. Often, these issues show up as isolated incidents that resolve on their own. However, in the worst cases, they can cause false positives in your check results, where Checkly might report that your site or server is slow or unavailable, even though you know it's up and running.
+
+In this section, we’ll take a closer look at why these intermittent issues can occur and how you can protect your monitoring setup from them.
+
+## Common Causes of Performance Variations
+
+Networking is complex, and hiccups happen. These can occur anywhere—within our infrastructure, on your end, or somewhere in between—and don’t necessarily mean that your server or site is actually down. Here are some common factors that can affect performance:
+
+- **Network Latency:** The time it takes for data to travel between points. This can vary due to distance, routing, or congestion.
+- **Packet Loss:** Sometimes data packets don’t make it to their destination and need to be resent, causing delays.
+- **Server Response Time:** Both our servers and yours might occasionally take a bit longer to respond.
+- **Infrastructure Issues:** Temporary problems like routing errors, server load spikes, or scheduled maintenance.
+
+While these factors are generally well-managed, they can’t be completely avoided. However, you can take steps to make your monitoring setup more resilient.
+
+## What Is an Accepted Failure Rate?
+
+<!--- This section would need to be refined more if we want to keep it i.e. specifiy which specific metrics we are talking about --->
+
+At Checkly, we understand that perfect infrastructure performance is unrealistic. That’s why we define an "accepted failure rate," which is a small percentage of checks that might be impacted by by server- and network anomalies.
```

Comment:
we need to define better what that means, otherwise people will ask questions. Does it mean amount of checks failing? requests? packets?

Reply:
For sure, I was hoping you could help me specify which exact metrics we should be talking about here (and possibly align these with internally set up SLOs). Happy to discuss at our next 1:1

Reply:
"0,002% total checks that run on checkly are false positives / abnormal check results -> Weder mit Checkly noch dem Check target zu tun haben -> von x checks ist y fehlerhaft -> check frequency von 1 min, entspricht das einem, alle x tausend mal kann es passiern dass ein issue passiert, failed check per woche"